.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve.
+labels: bug
+---
+
+**Description**
+
+A clear and concise description of what the bug is.
+
+* Tarantool version (`tarantool --version`):
```

Comment:
Maybe it worth to add `| head -n 1`.

Reply:
Or leave just 'Tarantool version:'.

Reply:
Sometimes it's necessary to see the build flags, so IMHO stripping this info is inconvenient for postmortem analysis.

Reply:
`tarantool --version` contains flags, major, minor versios and part of commit hash, target and compiler. I think that all that information is valuable for us.

Reply:
So it should be like the following:

```
<details>
<summary>Tarantool version</summary>

<place `tarantool --version` output here>
</details>

OS: ...
```

The separation from the item-per-line `key: value` list is because the output is expected to be multiline. The `<details/>` tag is because the output is relatively big and mostly the similar.

Reply:
discussed in private conversation, and now it looks like this https://gist.github.com/ligurio/8d8875b7c283b4f7ebdc103e1c777fe8

Reply:
> The separation from the item-per-line `key: value` list is because the output is expected to be multiline.

It was resolved.

> The `<details/>` tag is because the output is relatively big and mostly the similar.

It was declined by Sergey.

Okay, let's try and look.